doc/conf.py

```diff
@@ -53,18 +46,18 @@
 master_doc = 'index'
 
 # General information about the project.
-project = u'MACHtutorial'
-copyright = u'2018, MDOLab'
-author = u'MDOLab'
+project = 'MACH-Aero Tutorial'
+copyright = '2020, MDOLab'
+author = 'MDOLab'
```

Comment:
In some cases we have `MDO Lab`, which I believe is what Prof. Martins prefers.
In any case we should standardize this across repos.